swagger.yaml

```diff
@@ -133,12 +133,14 @@ paths:
           required: false
           schema:
             type: "string"
+            format: "date-time"
```

Comment:
Hm. I don't think that quotes are needed here. At least the rest of the file has types without quotes...